### PR TITLE
Move bookmark link to post header

### PR DESCRIPTION
**Pull request by jcmustin**
This PR brings the bookmark link out of the post page actions dropdown and into the post header, while also making a few other styling changes to the secondary info section, in accordance with designs [here](https://www.figma.com/file/OD4vJlLuWriHqG1WJXY4LV/Q1.1-2023?node-id=127%3A2846&t=HfpqlBXA1NEuCe2E-0).
- Use a comment icon rather than "n comments"
- Remove ordinal from date (8th Feb 2023 -> 8 Feb 2023)
- Increase secondary info font size to unify styling with author name
- Make spacing before triple dot icon the same as other secondary info items
- Increase margin between secondary info and title on EA Forum to 16px (site-specific because the original margin was already site-specific)
- Minor modifications to menu item styling on the EA Forum

## Desktop
image

## Mobile
image

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1203930944625756) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by agnesstenlund** [comment]
This looks really really great @jcmustin 👏  thanks for the clear PR description too.

I only have one tiny design nit-pick: looks like the tooltip appears on the right, whereas it otherwise seems to appear below

https://user-images.githubusercontent.com/13235378/217773409-30c79ca7-158b-4028-9790-eaba821c65ac.mov


Also looks like we haven't yet made the naming change in the user menu + on the Frontpage (guessing this was intentional as it wasn't completely clear if we were moving from Bookmark to Saved)

Screenshot 2023-02-09 at 10 31 02
Screenshot 2023-02-09 at 10 30 35
Screenshot 2023-02-09 at 10 37 19

**Review by s-cheng — CHANGES_REQUESTED** [review]
Looks great, thanks!

1. Please address Agnes's comments
2. Last I saw on slack, LW wanted this to be forum-gated right? If that's no longer the case, then this needs some styling tweaks before it's ok to ship to LW. (Either way, please make sure to test this with `forumType` = LessWrong.)

LW current styling:
Screen Shot 2023-02-13 at 4 01 45 PM

On this branch (prob just need to adjust the icon size?):
Screen Shot 2023-02-13 at 4 01 09 PM

File: packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx [hunk 1]
```diff
@@ -229,13 +231,13 @@ const PostsPagePostHeader = ({post, answers = [], toggleEmbeddedPlayer, hideMenu
               </NewFeaturePulse>
             )
           }
-          <div className={classes.commentsLink}>
+          {post.startTime && <div className={classes.commentsLink}>
             <AddToCalendarButton post={post} label="Add to Calendar" hideTooltip={true} />
-          </div>
+          </div>}
```
Comment (s-cheng) [inline packages/lesswrong/components/posts/PostsPage/PostsPagePostHeader.tsx:236]: This button's font size is too small, plus we prob want to adjust the spacing between the icon and the label to be consistent:
Screen Shot 2023-02-13 at 4 08 37 PM

**Comment by jpaddison3** [comment]
@s-cheng, this should be assigned to you right?

**Review by s-cheng — APPROVED** [review]

**Comment by s-cheng** [comment]
@agnesstenlund I'm gonna call this done for now, can you do a review? I think almost all the things from your figma are implemented, the main thing missing is the hover still decreases opacity instead of making the color darker, but I think we should just do that in another PR so we can finally merge this one in.

**Pull request merged**